### PR TITLE
Start 2022 election process.

### PR DESCRIPTION
**Pull request by craigbox**
Initial preparation for 2022 steering committee Community Seat election.

### CODE REVIEW

File: steering/elections/2022/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+- Providing neutral mediation as appropriate to try to resolve non-technical disputes that arise as part of the project.
+
+## Schedule
+
+| Date               | Event          |
+|--------------------|----------------|
+| September 19       | Announcement of Election and publication of VOTERS.md |
+| October 2          | Candidate bios and voting exception forms due by 2359 UTC (5pm PDT) |
+| ~1 week            | Election prep week (VOTERS.md validation, Elekto setup and testing) |
+| October 3          | Election begins via Elekto UI |
```
Comment (hzxuzhonghu) [inline steering/elections/2022/README.md:24]: Note from October 1 to October 7, Chinese people will be OOO for National Holiday

Comment (craigbox) [reply steering/elections/2022/README.md:24]: Thanks Zhonghu.  That still leaves almost the whole nomination period, and then allows Oct 7 onwards to vote.  Are you happy to progress with these dates?

Comment (hzxuzhonghu) [reply steering/elections/2022/README.md:24]: thanks, SGTM

**Review by irisdingbj — APPROVED** [review]
Thanks, LGTM

**Pull request merged**